pages/.eleventy.js

```diff
@@ -1,8 +1,21 @@
 const fs = require('fs');
 const chalk = require('chalk');
 const {isDev} = require('./11ty/env.config');
+const {sort} = require('./views/_data/render-docs');
+const typeDoc = require('./docs/typeDoc/typedoc');
+const {parseJSON} = require('./docs/typeDoc/parser/typedoc-json-parse');
 
 module.exports = (config) => {
+
+  const renderDock = async(entryPoints, output, callback) => {
+    await typeDoc(entryPoints, output);
+    const json = require('./' + output);
+    const data = sort(json);
+    callback(null, parseJSON(data));
+  }
+
+  config.addNunjucksAsyncFilter('generateDoc', renderDock);
+
```

Comment:
Any changes for the ` .eleveny.js` file is deprecated in our project configuration.
All of the 11ty build extendsion should be in bounds of `11ty` folder